package.json

```diff
@@ -49,6 +49,7 @@
     "dependencies": {
         "@js-joda/core": "^3.2.0",
         "catbuffer-typescript": "^1.0.0",
+        "js-sha3": "^0.8.0",
         "long": "^4.0.0",
         "ripemd160": "^2.0.2",
         "sha3": "^2.1.4",
```

Comment:
why two different sha3 libs?
I assume sha3 should have _both_ (especially given that it's used as 'keccak' hasher)

Reply:
sure, let me update those `js-sha3` to `sha3`

Reply:
Thanks! Travis is complaining about the missing 'js-sha3'

https://app.travis-ci.com/github/symbol/sdk-typescript/jobs/527443710#L274

I would update the Travis pr once this one is merged...

https://github.com/symbol/sdk-typescript/pull/24